lib/src/simple_op_store.rs

```diff
@@ -344,6 +362,7 @@ fn ref_target_to_proto(value: &RefTarget) -> Option<crate::protos::op_store::Ref
 }
 
 fn ref_target_from_proto(maybe_proto: Option<crate::protos::op_store::RefTarget>) -> RefTarget {
+    // TODO: Delete legacy format support in 0.10+
```

Comment:
Removed `in 0.10+` from the comment. Since we don't run one-off format conversion, old views are more likely accessed by new jj binaries.